src/shadowbox/server/manager_metrics.ts

```diff
@@ -15,8 +15,26 @@
 import {PrometheusClient} from '../infrastructure/prometheus_scraper';
 import {DataUsageByUser, DataUsageTimeframe} from '../model/metrics';
 
+export type TunnelTimeDimension = 'access_key' | 'country' | 'asn';
+
+interface TunnelTimeRequest {
+  time_window: {
```

Comment:
Let's define a Duration type and reuse

Reply:
It would be:

```ts
interface Duration {
   seconds: number;
}
```

Only one field. This makes the type reusable, and clearly marks all the variables that take a duration.